packages/webview_flutter/webview_flutter_wkwebview/pubspec.yaml

```diff
@@ -16,13 +16,17 @@ flutter:
         pluginClass: FLTWebViewFlutterPlugin
 
 dependencies:
+  device_info_plus: ^3.1.1
```

Comment:
I definitely don't want to do this. We should build the specific functionality we need into the plugin.